src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -193,6 +193,7 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 	<AndroidAotMode Condition=" '$(AndroidAotMode)' == '' ">None</AndroidAotMode>
 	<AotAssemblies Condition=" '$(AndroidAotMode)' != '' And '$(AndroidAotMode)' != 'None' And '$(AndroidAotMode)' != 'Interpreter' ">True</AotAssemblies>
 	<AotAssemblies Condition=" '$(AotAssemblies)' == '' ">False</AotAssemblies>
+    <_AndroidAotStripLibraries Condition=" '$(_AndroidAotStripLibraries)' == '' And '$(AndroidIncludeDebugSymbols)' != 'true' ">True</_AndroidAotStripLibraries>
```

Comment:
One issue is `$(AndroidIncludeDebugSymbols)` is defined later in the file, ~line 310, so it would be blank here.

Reply:
Also indentation :D

Reply:
> Also indentation :D

I still say we should unify indentation in MSBuild files :) `.editorconfig` says it should be two spaces, and that's what my editor used here... :) But the rest of the lines above is indented with tabs... :)